react/features/speaker-stats/components/native/SpeakerStatsList.js

```diff
@@ -0,0 +1,102 @@
+// @flow
```

Comment:
most of the functionality seems to be taken from https://github.com/jitsi/jitsi-meet/blob/master/react/features/speaker-stats/components/SpeakerStats.js. Maybe we can find a way to reuse it, e.g. create an AbstractSpeakerStatsList that contains the common code.